Gemfile

```diff
@@ -13,7 +13,14 @@ gem 'bundler'
 gem 'rake'
 
 group :system_tests do
-  gem 'beaker'
+  beaker_version = ENV.fetch('BEAKER_VERSION', ['>= 4.17.0', '< 5.0.0'])
+
+  if "#{beaker_version}".include?('/')
+    gem 'beaker', :source => beaker_version
```

Comment:
Using `:source` will let folks specifcy an alternative rubygems repo, but it won't let them point to an arbitrary git repo or ref (which in this instance seems like the more likely use case).